### ISSUE TITLE
Setting limits to number fields not saving

### ISSUE THREAD

**Issue by peterurban** [verbb/formie#1077]
### Describe the bug

Setting limits on number fields is not being saved with the form. 
When saving the form, the "Limit numbers" toggle is turned off again. 

No errors are shown in the UI or any of the logs.

### Steps to reproduce

1. Open form
2. Selecting Number field and opening settings
3. Enabling "Limit numbers"
4. Set min and max values
5. Click Apply
6. Save form

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

### Craft CMS version

4.2.3

### Plugin version

2.0.9

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/formie/releases/tag/2.0.10)
